src/python/WMCore/WMSpec/StdSpecs/Resubmission.py

```diff
@@ -39,15 +39,12 @@ def buildWorkload(self, arguments):
         helper.ignoreOutputModules(self.ignoredOutputModules)
 
         # override a couple of parameters, if provided by user
-        if 'Memory' in arguments:
-            helper.setMemory(arguments['Memory'])
-        if 'Campaign' in arguments and not helper.getCampaign():
-            helper.setCampaign(arguments["Campaign"])
         if 'RequestPriority' in arguments:
             helper.setPriority(arguments["RequestPriority"])
-        if 'TimePerEvent' in arguments:
-            for task in helper.taskIterator():
-                task.setJobResourceInformation(timePerEvent=arguments["TimePerEvent"])
+        if arguments['OriginalRequestName'] != 'TaskChain' or isinstance(arguments['Memory'], dict):
+            helper.setMemory(arguments['Memory'])
```

Comment:
And this guy sets Memory for ALL tasks. I'll have to filter only Production/Processing task types to have this set, do you agree @ticoann ?